hifireg/registration/views/registration.py

```diff
@@ -280,11 +280,15 @@ def make_payment(request):
     form = None
     if request.method == 'POST':
         if 'previous' in request.POST:
-            return redirect('register_misc')
+            # return redirect('register_misc')
+            return redirect('hello_world')
         return redirect('payment_confirmation')
     else:
         return render(request, 'registration/payment.html', {'form': form})
 
 
 def payment_confirmation(request):
     return render(request, 'registration/payment_confirmation.html', {})
+
+def hello_world(request):
+    return render(request, 'registration/base.html', {})
```

Comment:
You want to use a CBV here instead.
```suggestion
class PaymentPlan(View):
    def get(self, request):
        return render(request, 'registration/base.html', {})

```
but that's still using "base.html" directly. 
What you want to do is inherit from "base.html". So you'll need to make a new template file.
Here is a resource for "django template inheritance":
https://docs.djangoproject.com/en/3.0/ref/templates/language/#template-inheritance